manifests/init.pp

```diff
@@ -159,13 +159,15 @@
   if $fpm {
     Anchor['php::begin'] ->
       class { '::php::fpm':
+        ensure           => $ensure,
```

Comment:
if `fpm.pp` is only ever declared from `init.pp` should it even have defaults for it's parameters?  The defaults never get used, do they?